### PR TITLE
Rhino config

### PR DESCRIPTION
**Pull request by jaredgalloway**
Adding rhino configurations.

### CODE REVIEW

File: modules/treetime_validation.nf [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 nextflow.enable.dsl = 2
 
-params.subtrees = Channel.of(20, 50, 100, 200, 500, 750, 1000, 1250, 1500, 2000)
+params.subtrees = Channel.of(20, 50, 100, 200, 500, 750, 1000)//, 1250, 1500, 2000)
```
Comment (jaredgalloway) [inline modules/treetime_validation.nf]: @4ment I only ran with the smaller parameter inputs. I can change this line to include the full range of parameters. And once we merge, then you can try running it yourself?

**Comment by jaredgalloway** [comment]
@4ment The pipeline has run successfully. As far as I can tell from our slack conversations there are two more things you would like:

1. for the entire pipeline to be run on a single node (not just each job, AFAICT)
3. For dynamic allocation of resources. 

I think both of these can be achieved, but maybe we can finish up this PR, make sure it's running for you on our cluster. And then discuss these (1 & 2 above) separately? I'll happily open separate issues and Submit PR's for these after this is merged

File: configs/rhino.config [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+profiles {
+    rhino {
+        process {
+            executor = 'slurm'
+            scratch = '/fh/scratch/delete30/matsen_e/mathieu/temp/nf-scratch/'
+            queue = 'campus-new'
+            cpus = 3
+            memory = 20.GB
+            container = 'quay.io/matsengrp/autodiff-experiments:rhino_config'
```
Comment (4ment) [inline configs/rhino.config]: Please change it to 4ment/autodiff-experiments
```suggestions
container = '4ment/autodiff-experiments'
```

File: modules/treetime_validation.nf (hunk 1 above)
Comment (4ment) [reply modules/treetime_validation.nf]: Sounds good, please include full range

File: modules/macro_flu.nf [hunk 3]
```diff
@@ -91,6 +91,7 @@ process RUN_PHYSHER {
 
   publishDir "$params.results/macro/physher", mode: 'copy'
 
+
```
Comment (4ment) [inline modules/macro_flu.nf]: Remove extra space please

**Review by 4ment — CHANGES_REQUESTED** [review]
Everything looks great, I have added some comments and can you also remove the r_ouput/nextflow_report.html file.

File: configs/rhino.config (hunk 2 above)
Comment (jaredgalloway) [reply configs/rhino.config]: Hm, so for whatever reason your image was not working for me. Although it looks like it was re-build recently so I'll try again. 

however, a quay build using [your dockerfile](https://github.com/4ment/autodiff-experiments/blob/master/Dockerfile) - worked on my trial run.

I'll follow up here

**Comment by jaredgalloway** [comment]
@4ment I've updated according to your comments. I'm re-running now and your updated dockerhub image seems to be working now. I'll follow up when it completes so we can validate, and merge. :rocket:

**Pull request merged**